### DISCUSSION TITLE
Add info and disclaimers to ticket report page.

### DISCUSSION THREAD

**Discussion by televandalist** [RetroAchievements/RAWeb#805] [Ideas]
There really should be some info on the ticket report page rather than just the fields for the report itself. 

- Information on finding and reporting the correct hash, emulator versions, etc. and why it's important. Not everyone is a supernerd, after all. 
- Disclaimer saying "Doesn't work" is about the most worthless thing you can add to a ticket with a "Doesn't trigger" issue. These are known as the "No shit?" tickets. Maybe word it more politely than that. 

Overall, it should all have a "help us help you" approach. Better tickets lead to less disgruntled devs, which leads to more and better achievements, which leads to happiness and rainbows for everyone.

**Comment by jeffjenx** [comment]
Do we have any technical writers to provide the content for this material? It should be a relatively easy implementation, but I'm not entirely familiar with the reporting process so I would need the content to be written for me.

**Comment by Jamiras** [comment]
Implemented in #674. The information is in a popup. The user still has to click the "?" to see the additional information.

There's also some minimalist matching code to prevent simply submitting "Doesn't work" as a description.
